### PR TITLE
Prep `MultipleMerkleDistributor` for OP Trading Rewards Distribution

### PR DESCRIPTION
**Pull request by JChiaramonte7**
In preparation for OP trading rewards we need a MultipleMerkleDistributor contract that plainly sends out tokens instead of locking them in escrow.

### Changes
- Renaming of **MultipleMerkleDistributor** (old, existing contract) -> **EscrowedMultipleMerkleDistributor**
- New **MultipleMerkleDistributor** which has escrow functionality removed

@JaredBorders @tommyrharper This PR is mostly renaming of files, but see the diff of `MultipleMerkleDistributor.sol` for an overview of the changed functionality. 

image

### CODE REVIEW

**Review by JaredBorders — CHANGES_REQUESTED** [review]
Most comments are for pretty small changes. I can't think of an instance where the token transfer would ever fail, but if the token defined was not OP nor KWENTA it could.

File: contracts/EscrowedMultipleMerkleDistributor.sol [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    function claim(
+        uint256 index,
+        address account,
+        uint256 amount,
+        bytes32[] calldata merkleProof,
+        uint256 epoch
+    ) public override {
+        require(
+            !isClaimed(index, epoch),
+            "EscrowedMultipleMerkleDistributor: Drop already claimed."
```
[the remark this answers is not in the document]
Comment (JChiaramonte7) [reply contracts/EscrowedMultipleMerkleDistributor.sol:96]: Tried to refrain from making too many changes if they weren't necessary so I'm gonna leave this one as is

File: scripts/deploy-multiple-merkle-distributor.ts [hunk 2]
```diff
@@ -13,8 +12,7 @@ async function main() {
 
     const tradingRewards = await deployMultipleMerkleDistributor(
         deployer,
-        KWENTA_ADDRESS,
-        REWARD_ESCROW_ADDRESS
+        KWENTA_ADDRESS
```
[the remark this answers is not in the document]
Comment (JChiaramonte7) [reply scripts/deploy-multiple-merkle-distributor.ts:15]: Thank you good catch, was going to use this script, but didn't run the deploy yet

File: contracts/MultipleMerkleDistributor.sol [hunk 3]
```diff
@@ -100,14 +91,9 @@ contract MultipleMerkleDistributor is IMultipleMerkleDistributor, Owned {
...
-        // mark it claimed and send the token to RewardEscrow
+        // mark it claimed and send the token
         _setClaimed(index, epoch);
-        IERC20(token).approve(rewardEscrow, amount);
-        IRewardEscrow(rewardEscrow).createEscrowEntry(
-            account,
-            amount,
-            52 weeks
-        );
+        IERC20(token).transfer(account, amount);
```
[the remark this answers is not in the document]
Comment (JChiaramonte7) [reply contracts/MultipleMerkleDistributor.sol]: Ended up using safeTransfer instead

File: scripts/deploy-multiple-merkle-distributor.ts [hunk 4]
```diff
@@ -26,16 +24,14 @@ async function main() {
 
 async function deployMultipleMerkleDistributor(
     owner: SignerWithAddress,
-    kwenta: string,
-    rewardEscrow: string
+    kwenta: string
```
Comment (JaredBorders) [inline scripts/deploy-multiple-merkle-distributor.ts:27]: this should not be kwenta

File: scripts/deploy-multiple-merkle-distributor.ts [hunk 5]
```diff
@@ -26,16 +24,14 @@ async function main() {
...
 ) {
     const MultipleMerkleDistributor = await ethers.getContractFactory(
         "MultipleMerkleDistributor"
     );
     const multipleMerkleDistributor = await MultipleMerkleDistributor.deploy(
         owner.address,
-        kwenta,
-        rewardEscrow
+        kwenta
```
Comment (JaredBorders) [inline scripts/deploy-multiple-merkle-distributor.ts:34]: not kwenta

File: scripts/deploy-multiple-merkle-distributor.ts [hunk 6]
```diff
@@ -49,7 +45,7 @@ async function deployMultipleMerkleDistributor(
 
     await verify(
         multipleMerkleDistributor.address,
-        [owner.address, kwenta, rewardEscrow],
+        [owner.address, kwenta],
```
Comment (JaredBorders) [inline scripts/deploy-multiple-merkle-distributor.ts:48]: not kwenta

**Review by JaredBorders — APPROVED** [review]
LGTM after you change the names in the deploy script 👍

File: scripts/deploy-multiple-merkle-distributor.ts
Comment (JaredBorders) [inline scripts/deploy-multiple-merkle-distributor.ts:1]: 👍

**Pull request merged**